plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithMultinodeQueryRunner.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.metastore.cache;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.hive.HiveQueryRunner;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.QueryRunner;
+import org.testng.annotations.Test;
+
+import static io.trino.tpch.TpchTable.NATION;
+
+@Test(singleThreaded = true)
+public class TestCachingHiveMetastoreWithMultinodeQueryRunner
+        extends AbstractTestQueryFramework
+{
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        return HiveQueryRunner.builder()
+                .setHiveProperties(ImmutableMap.of("hive.metastore-cache-ttl", "60m"))
+                // only so that tpch schema is created
+                .setInitialTables(ImmutableList.of(NATION))
```

Comment:
We should fix the query runner instead of hacking around our things that are supposed to serve us.

This is an issue since 5bad95f1db1945df297a532558d12b0500a657ce cc @kokosing .
Perhaps in https://github.com/trinodb/trino/blob/d3b0c523b378ac72197e0f06110437ac9d7f0de6/plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java#L235 we could initialize session to use a "default" schema (needs to be created) when `initialTables.isEmpty()`. However, it seems to me the `initialTables` would control more than the name suggests, which isn't great. In my opinion this approach would not be better then reverting that commit, but I probably don't see the full picture here.

To avoid blocking PR on something not material, please file an issue than `HiveQueryRunner` with no initial tables apparently cannot be used and link to it with a TODO here.